app/src/main/res/layout/activity_main.xml

```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".ui.meme_it_up.MemeItUpActivity">
```

Comment:
do not use **_** to separate words in packages, use **.**

Reply:
done, thanks